packages/blockchain-wallet-v4-frontend/src/data/components/debitCard/sagas.ts

```diff
@@ -7,14 +8,30 @@ import { selectors } from 'data'
 import { actions as A } from './slice'
 
 export default ({ api }: { api: APIType }) => {
+  const getCards = function* () {
+    try {
+      const data = yield call(api.getDCCreated)
+      yield put(A.getCardsSuccess(data))
+    } catch (e) {
+      console.error('Failed to get account cards', errorHandler(e))
+      yield put(A.getCardsFailure())
+    }
+  }
+
   const getProducts = function* () {
     const debitCardModuleEnabled = (yield select(
       selectors.core.walletOptions.getWalletDebitCardEnabled
     )).getOrElse(false)
     if (debitCardModuleEnabled) {
       try {
-        const data = yield call(api.getDCProducts)
-        yield put(A.getProductsSuccess(data))
+        const products = yield call(api.getDCProducts)
```

Comment:
same here. why no Loading remote state?

Reply:
same response I guess 😅 
This does not render something after request is done. I left error actions for showing some kind of error message to the user in the future